kubernetes-templates/cassandra/0/rancher-compose.yml

```diff
@@ -2,3 +2,9 @@
   name: "K8s Example Cassandra"
   version: "1.0.0"
   description: ""
+  questions:
```

Comment:
That's odd, I tested again and it showed up for me. I merged this into my master branch moments ago, could you try again?